packages/rest/test/acceptance/sequence/sequence.acceptance.ts

```diff
@@ -103,14 +108,14 @@ describe('Sequence', () => {
       response.statusCode = 418; // I'm a teapot
       response.end();
     };
-    app.bind(SequenceActions.REJECT).to(reject);
+    server.bind(SequenceActions.REJECT).to(reject);
 
     return whenIMakeRequestTo(app).get('/unknown-url').expect(418);
   });
 
   it('makes ctx available in a custom sequence handler function', () => {
-    app.bind('test').to('hello world');
-    app.handler((sequence, request, response) => {
+    server.bind('test').to('hello world');
```

Comment:
I am proposing to keep `test` bound at application level. Conceptually, it's some application-wide config used by a route, at least that's how I see it.